documentPage/doc.css

```diff
@@ -0,0 +1,78 @@
+@import url(../landingpage.css);
+
+.doc-container{
+    display: grid;
+    padding: 0px 10rem;
```

Comment:
padding : 0rem 10rem or 0 10rem could be written

Reply:
did the changes 👍